features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Base/Header/Header.js

```diff
@@ -166,10 +167,13 @@ class Header extends React.Component {
                                     <SearchIcon />
                                 </IconButton>
                                 {/* API add menu */}
```

Comment:
this comment needs to move to the bottom of scope validator

Reply:
This is fixed with https://github.com/wso2/carbon-apimgt/pull/5147